src/main/java/core/basesyntax/service/handler/impl/PurchaseOperationHandler.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.handler.impl;
+
+import core.basesyntax.service.handler.OperationHandler;
+
+public class PurchaseOperationHandler implements OperationHandler {
+    @Override
+    public Integer operate(Integer oldValue, Integer transactionValue) {
+        return transactionValue - oldValue;
```

Comment:
What will it be if your transactionValue is less than oldValue? I believe that you need to fix it.